arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -98,11 +98,15 @@
 
 #include "IResearch/IResearchView.h"
 
+#include <regex>
+
 using namespace arangodb;
 using namespace arangodb::application_features;
 using namespace arangodb::options;
 
 namespace arangodb {
+  
+std::regex const journalFileRegex("^\\d+\\.log$", std::regex::ECMAScript);
```

Comment:
do we really need a regex to match for suffix .log ?

Reply:
Not really. However, it is only used for looping over the logfiles at startup.
I can turn this into a simpler string operation in devel.